packages/core/src/plugins/amqp/router/adapter.ts

```diff
@@ -3,8 +3,9 @@ import get = require('get-value')
 import is = require('is')
 import noop = require('lodash/noop')
 import { ActionTransport } from '../../..'
-import { ServiceRequest } from '../../../types'
 import { Router } from '../../router/factory'
+import { kReplyHeaders } from '@microfleet/transport-amqp/lib/constants'
```

Comment:
lets update @microfleet/transport-amqp and add symbol as exports
something like `symbols.replyHeaders`